superset/assets/src/dashboard/containers/Chart.jsx

```diff
@@ -36,6 +38,7 @@ function mapStateToProps(
     charts: chartQueries,
     dashboardInfo,
     dashboardState,
+    dashboardLayout: undoableLayout,
```

Comment:
What is `undoableLayout`?

Reply:
I have used the same name as used by Superset at [this location](https://github.com/Guavus/incubator-superset/blob/master/superset/assets/src/dashboard/containers/DashboardHeader.jsx#L54) to be consistent